ofrak_core/test_ofrak/components/test_uefi_component.py

```diff
@@ -61,12 +61,12 @@ async def unpack(self, root_resource: Resource):
         await root_resource.unpack_recursively()
 
     async def get_descendants_to_verify(self, unpacked_root_resource: Resource) -> Dict:
-        result = {
-            await (
-                await descendent.view_as(FilesystemEntry)
-            ).get_path(): await descendent.get_data()
-            for descendent in await unpacked_root_resource.get_descendants()
-        }
+        result = {}
+        for descendent in await unpacked_root_resource.get_descendants():
+            if descendent.has_tag(FilesystemEntry):
+                result[
+                    await (await descendent.view_as(FilesystemEntry)).get_path()
+                ] = await descendent.get_data()
```

Comment:
```suggestion
        for descendant in await unpacked_root_resource.get_descendants():
            if descendant.has_tag(FilesystemEntry):
                result[
                    await (await descendant.view_as(FilesystemEntry)).get_path()
                ] = await descendant.get_data()
```

Minor typo in the original we might as well fix here.